UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileShareService.cs

```diff
@@ -310,6 +314,96 @@ public async Task<string> SearchReadMeFilePath(string batchId,string correlation
             return filePath;
         }
 
+        public bool CreateZipFileForExchangeSet(string exchangeSetZipRootPath, string correlationId)
+        {
+            var zipName = $"{exchangeSetZipRootPath}.zip";
+            string path = Path.Combine(exchangeSetZipRootPath, zipName);
+            if (fileSystemHelper.CheckDirectoryAndFileExists(exchangeSetZipRootPath, path))
+            {
+                fileSystemHelper.CreateZipFile(exchangeSetZipRootPath, zipName);
+
+                logger.LogInformation(EventIds.CreateZipFileRequestCompleted.ToEventId(), "Exchange set V01X01.zip created with _X-Correlation-ID:{correlationId}", correlationId);
```

Comment:
I'm not sure about this one but will the file always be called V01X01.zip or would we better off passing zipName into the log file?

i.e. should this be "Exchange set {zipName} created........."

Reply:
Accessed zipName from config file.